src/components/anchor-navigation/anchor-navigation.component.js

```diff
@@ -0,0 +1,133 @@
+import React, {
+  useState, useRef, useEffect, useCallback
+} from 'react';
+import PropTypes from 'prop-types';
+import throttle from 'lodash/throttle';
+
+import Event from '../../utils/helpers/events';
+import { StyledAnchorNavigation, StyledNavigation, StyledContent } from './anchor-navigation.style';
+import AnchorNavigationItem from './anchor-navigation-item.component';
+
+const AnchorNavigation = ({ children, stickyNavigation, styleOverride }) => {
+  const [selectedIndex, setSelectedIndex] = useState(0);
+
+  const sectionRefs = useRef(React.Children.map(stickyNavigation.props.children, child => child.props.target));
+
+  const anchorRefs = useRef(Array.from({
```

Comment:
This will need a polyfill or different implementation while we still support `IE` 😢  
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from

Reply:
True, I forgot about IE 11 :)

Reply:
It seems that we dont need to support IE11 at all anymore

Reply:
🎉